src/systemTest/java/com/flightstats/hub/testcase/WebhookLifecycleTest.java

```diff
@@ -172,11 +192,11 @@ private String addItemToChannel(Object data) {
         return response.body().get_links().getSelf().getHref();
     }
 
-    private Webhook buildWebhook(String startItem) {
+    private Webhook buildWebhook(String startItem, int parallelCalls) {
         return Webhook.builder()
-                .channelUrl(retrofit.baseUrl() + "channel/" + channelName)
-                .callbackUrl(callbackServer.getUrl())
-                .parallelCalls(2)
+                .channelUrl(getHubClientBaseUrl() + "channel/" + channelName)
+                .callbackUrl(getCallbackClientBaseUrl() + "callback/")
+                .parallelCalls(parallelCalls)
                 .batch("single")
                 .startItem(isBlank(startItem) ? null : startItem)
```

Comment:
Did you decide to skip adding a `@Wither` annotation because it makes the API of the `Webhook` class ugly? I would rather explicitly override the start item in the test because going down the path of adding params gets hard to read. Having a method that sets some defaults and relies on the calling test to specify the fields that are important for each test.

Reply:
Added @Wither annotation